console/src/test/java/org/web3j/console/project/TemplateProviderTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 Web3 Labs LTD.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.console.project;
+
+import java.io.IOException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TemplateProviderTest {
+
+    TemplateProvider templateProvider;
+
+    {
```

Comment:
this doe not look right.. a random code block in the code? How does this not fail to compile?

this should be in a `@Before` method

Reply:
done